src/extensions/remove-image/remove-image.cypress.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Include our constants
+ */
+import * as helpers from '../../../.dev/tests/cypress/helpers';
+
+describe( 'Test CoBlocks Replace Image extension', function() {
+	/**
+	 * Setup Gallery data
+	 */
+	const imageData = {
+		fileName: '150x150.png',
+		fileBase: '150x150',
+		pathToFixtures: '../.dev/tests/cypress/fixtures/images/',
+	};
+
+	/**
+	 * Test that we can add a image block to the content add image,
+	 * and alter image using the Replace Image extension
+	 */
+	it( 'Test core/image block extends with Replace Image component.', function() {
+		const { fileName, fileBase, pathToFixtures } = imageData;
+		helpers.addCoreBlockToPage( true, 'image' );
+
+		cy.get( '.replace-image-button' ).should( 'not.exist' );
+
+		cy.fixture( pathToFixtures + fileName, 'base64' ).then( ( fileContent ) => {
+			cy.get( 'div[data-type="core/image"]' )
+				.find( 'div.components-drop-zone' ).first()
+				.upload(
+					{ fileContent, fileName, mimeType: 'image/png' },
+					{ subjectType: 'drag-n-drop', force: true, events: [ 'dragstart', 'dragover', 'drop' ] },
+				)
+				.wait( 2000 ); // Allow upload to finish.
```

Comment:
From what I've read about the behavior of Cypress, we don't need to wait here arbitrarily. Cypress should be waiting by default for the upload to finish and move on as soon as it's done. I think you would use `.then()` to perform an action or assertion after upload if you need to.